packages/js/product-editor/src/components/variations-table/image-actions-menu/image-actions-menu.tsx

```diff
@@ -111,8 +111,9 @@ export function ImageActionsMenu( {
 					</MenuGroup>
 
 					<VariationQuickUpdateMenuItem.Slot
+						name=""
 						group={ 'image-actions-menu' }
-						onChange={ onChange }
+						onChange={ ( value ) => onChange( value ) }
```

Comment:
Same as the `DownloadsMenuItem` -> I believe with your change in `VariationActionsMenus` types this change is not necessary.

Reply:
Fixed with 6bcf79b8cf3d057364541118c6c4fe8f3b03dc8d

Reply:
Fixed with c7abee4122ddaadebe35f8debd91dd0243195e9e